### DISCUSSION TITLE
AutoARIMA forecast exogenous inputs

### DISCUSSION THREAD

**Discussion by dchou-rh** [autogluon/autogluon#3501] [General]
Since known covariates are passed through TimeSeriesPredictor, why are they also not passed into AutoARIMA when forecasting? https://github.com/autogluon/autogluon/blob/master/timeseries/src/autogluon/timeseries/models/local/statsforecast.py#L42. According to statsforecast documentation, this is supported https://nixtla.github.io/statsforecast/src/core/models.html#autoarima.forecast

**Comment by shchur** [comment]
Hi, in some benchmarking that we ran internally we found that including the covariates often does not improve the performance of ARIMA (or even has a slight negative effect). If we include the covariates, this would also slow down model training and increase the code complexity, so we decided not to enable covariates for this model. We rely on other models like DeepAR, TFT, or tabular-based models like LightGBM to work well in the presence of covariates.

**Comment by dchou-rh** [comment]
Okay understood. In our use case, we find covariates helpful, so this will likely cause us to move directly back to statsforecast or similar package.

Would it make sense to include your findings and discussion in the Model Zoo and/or API documentation? This was not made apparent until after digging through the code a bit more and discovering that the covariates were not being passed to the ARIMA model.

**Reply by shchur** [reply]
Thank you for the suggestion! I have created an issue to track progress on this feature (#3508). I cannot guarantee that we will include it by the v1.0 release, but we will consider adding it in the future.
